CMakeLists.txt

```diff
@@ -642,6 +692,11 @@ if(NOT SKIP_INSTALL_ALL)
 endif()
 
 if(AVIF_CODEC_LIBRARIES MATCHES vmaf)
+    # vmaf only impacts the avifenc and avifdec targets below, not the library avif target above.
```

Comment:
Super nit: impacts => affects

library avif => avif library

This comment should also mention the examples, right?  I.e., the avif library and examples targets.

Reply:
Done in https://github.com/AOMediaCodec/libavif/pull/1616.